examples/seed/stack.yaml

```diff
@@ -0,0 +1,14 @@
+resolver: lts-14.16
```

Comment:
You don't need this file if you have another `stack.yaml` in the root.
I know this has been auto-generated, but it's better to remove it to not confuse casual readers.